package.json

```diff
@@ -46,6 +47,6 @@
   "devDependencies": {
     "@sideway/address": "^4.1.2",
     "browserslist": "^4.16.4",
-    "react-scripts": "^4.0.3"
```

Comment:
What's the reason for downgrading for `react-scripts` dependency?
It may break something.

Reply:
My fault